spec/selinux_module_spec.rb

```diff
@@ -45,7 +45,7 @@
   end
 
   it 'installs the dependency packages' do
-    expect(chef_run).to(install_package('make, policycoreutils, selinux-policy-devel'))
+    expect(chef_run).to(install_package('install_policy_devel_packages'))
```

Comment:
you really want to hard code your expects. If they're just calling a method you're not testing anything